models/public/hrnetv2/hrnetv2.md

```diff
@@ -0,0 +1,110 @@
+# hrnetv2
```

Comment:
I'll recommend to rename model to `hrnet-v2` or even `hrnet-v2-w48` if `w48` suffix is quite meaningful.

Reply:
w48 is model size, for semantic segmentation there is no any other available (small models w18 and w32 used as backbone for facial landmarks recognition and pose estimation respectively, w48 is the most popular as I know). So it can be ommited, I think. But there is another important thing which should be reflected in the name, HRNetV2 is only half of this model, its encoder part. @alexmasterdi, please add info about decoder into model name